app/javascript/dashboard/assets/scss/_layout.scss

```diff
@@ -29,7 +30,7 @@ body {
   }
 }
 
-.banner + .app-wrapper {
+.banner+.app-wrapper {
```

Comment:
Is it trivial to move these styles to tailwind classes directly? If it is let's do it in a separate PR

Reply:
Sure we can do it in a separate PR, for now I just removed this line from this PR